.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,83 @@
+name: Docker Smoke Test
+
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+
+env:
```

Comment:
We should have a way to generate a .env with reasonable defaults but now that we don't how about adding default values to the version variables in the docker compose file, hardcode the user name and generate the password in the GHA?